ci/dockerfiles/golang-builder/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:bionic
+
+# gcc for cgo
+RUN apt-get update && apt-get install -y --no-install-recommends \
+		g++ \
+		gcc \
+		libc6-dev \
+		make \
+		pkg-config \
+                wget \
+                git \
+                ca-certificates \
+	&& rm -rf /var/lib/apt/lists/*
+
+ENV GOLANG_VERSION 1.11.5
```

Comment:
Hey, I think we could have some more flexibility making this an `ARG` instead.

For instance:

```
ARG GOLANG_VERSION=1.11.5
```

This would allow one to build the image for any golang version out there with `docker build --build-arg=GOLANG_VERSION=blabla`, something that we could support in `concourse/builder` in the future, allowing us to automate the build for this image based on Go releases.

Wdyt?